packages/cerebral/src/operators/merge.js

```diff
@@ -17,7 +17,9 @@ export default function (target, ...values) {
     }))
   }
 
-  merge.displayName = 'operator.merge'
+  merge.displayName = `operator.merge(${String(target)}, ${values.map((value) => {
+    return String(value)
```

Comment:
It is not the value, it is just the string representation of the tag... so basically displays 

```js
"state`foo.bar`" 
```

not the value :)